resources/js/features/root/layout/Navbar.tsx

```diff
@@ -0,0 +1,82 @@
+import {
+  A,
+  AnchorProps,
+  Location,
+  useLocation,
+  useParams,
+} from "@solidjs/router";
+import cx from "classnames";
+import { IconTypes } from "solid-icons";
+import { HiSolidInformationCircle, HiSolidUserGroup } from "solid-icons/hi";
+import { Component, For, Show } from "solid-js";
+import s from "./style.module.scss";
+
+export const Navbar: Component = () => {
+  const location = useLocation();
+  const params = useParams<{ facilityUrl?: string }>();
+
+  return (
+    <aside class={cx(s.sidebar)}>
+      <div class={cx("flex flex-row justify-between")}>
+        <img src="/img/memo_logo.svg" class="h-14" />
+        <img src="/img/cpd_children_logo.svg" class="h-12" />
+      </div>
+      <nav class="flex-1">
+        <Show when={params.facilityUrl}>
+          <For each={navbarItems}>
+            {(item, index) => (
+              <A
+                data-index={index()}
```

Comment:
Dlaczego to ustawiasz?

Reply:
komponent For wymaga, aby ustawić data-index dla każdego elementu

Reply:
to trochę podobne do key w react

Reply:
Hm, tam gdzie ja czytałem to nic o tym nie było więc nie tak do końca wierzę póki co :)
https://www.solidjs.com/tutorial/flow_for?solved

Reply:
faktycznie, dokumentacja nic nie mówi. W sumie teraz nie wiem skąd to wziąłem :D

Reply:
poprawione